src/plugins/template/tests/functional/skip_tests_config.cpp

```diff
@@ -131,6 +131,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(.*ReferenceTopKTest.*topk_min_sort_none)");
 #endif
 
+#if defined(__APPLE__) && defined(OPENVINO_ARCH_X86_64)
+    // CVS-64054
```

Comment:
this ticket is closed.
should we create another one or reopen existing?

Reply:
created new - CVS-120988